README.md

```diff
@@ -1,34 +1,56 @@
 
 ## How we're different than other meetups / focus groups
-While other meetup groups in Charlotte are focused on discussing and improving knowledge and skill in a particular area, we focus on interactions.
+We don't seek to differentiate ourselves. In fact, we desire to ally with other meetups and groups. Where we desire learning and teaching to happen, many other groups already serve in this role in many niches. We will seek to partner and help these groups while learning from them and building new opportunities where none exist. We seek to smooth the pipeline from the time a student first desires to pursue a technical career to the time they are first hired, help their skills grow throughout their career, and seek to encourage them to give back to those coming after in the form of mentorship, leadership, and employment.
+
+We encourage everything to be ongoing, encouraging relationships wherever they may be found.
+
+## Our Focuses
+
+### Apprenticeship
+We believe that the training and mentoring of new and junior tech talent is currently lacking and is the responsibility of the community at large. We seek to bring the community together to cultivate the talents of these rising new assets to the community. We seek to establish organizations to help bring these people together to learn as well as network with each other and the rest of the community at large
+
+### Mentoring
+We believe that it is important for the community to help bring those who are newer to the community into the community. To do this, we seek to gather those who have a strong desire to do this and, for those who are unsure of how to accomplish this themselves, build a framework and find a place for them to do so.
+
+### Training
+We believe that constant training is imperative to ALL those in the tech community. Technology is moving incredibly quickly and it is important that we can, as a community, continue to share knowledge and build our collective skills up.
+
+### Leadership
+We believe that leadership is rare and we seek to cultivate both by seeking out those who are already good leaders to learn from, but also by giving those among us who accept this calling a path to practice their leadership skills so that this community can serve the community into perpetuity.
+
+## Our Core Principles
+
+### Egalitarianism
+As one of the primary principles in our country's founding, we also believe that equal opportunity is important to this community as well.
+
+### Diversity
+Diversity isn't just a 'right' thing, it's smart biologically, politically, and financially - it's just smart business. As such, we also believe diversity is important to building a strong organization that will last.
```

Comment:
The problem with that is that these terms aren't universal and making them bullet points leaves them open to very wide interpretation. I'm definitely not defending my definitions here as I just filled them out, but we should actually define what we mean by these terms otherwise we'll end up with twisted forms of them floating around.